addons/web/models/base_document_layout.py

```diff
@@ -120,6 +120,11 @@ def _compute_logo_colors(self):
     @api.depends('report_layout_id', 'logo', 'font', 'primary_color', 'secondary_color', 'report_header', 'report_footer', 'layout_background', 'layout_background_image', 'company_details')
     def _compute_preview(self):
         """ compute a qweb based preview to display on the wizard """
+
+        if self.env['base.document.layout'].browse(self.id):
```

Comment:
Can you put a comment to explain this, seems a bit magical or hacky without a proper comment to explain why we need to do that